src/shogun/labels/RegressionLabelsEncoder.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Yuhui Liu
+ *
+ */
+#ifndef _RegressionLabelsEncoder__H__
+#define _RegressionLabelsEncoder__H__
```

Comment:
```suggestion
#define _REGRESSIONLABELENCODER__H__
```
could you capitalise these to stay consistent please?